src/prob/mld_block_robust.jl

```diff
@@ -0,0 +1,218 @@
+"""
+    solve_robust_block_mld(
+        data::Dict{String,<:Any},
+        model_type::Type,
+        solver;
+        N::Int, ΔL::Float64,
+        kwargs...
+    )::Dict{String,Any}
+
+Solves a robust (N scenarios and ±ΔL load uncertainty) multiconductor optimal switching (mixed-integer) problem using `model_type` and `solver`.
+The default number of scenarios is set to 2 with load uncertainty of ±10%.
+"""
+function solve_robust_block_mld(data::Dict{String,<:Any}, model_type::Type, solver; N::Int=2, ΔL::Float64=0.1, kwargs...)::Dict{String,Any}
```

Comment:
i suggest to create a new function also called `solve_robust_block_mld` which includes lines 17-55, that accepts load scenarios generated by the user, i.e., `load_factor`.

For example:

```julia
function solve_robust_block_mld(data, load_scenarios, model_type, solver; kwargs...)
    # lines 17-55
end

function solve_robust_block_mld(data, model_type, solver; N=2, ΔL=0.1, kwargs...)
    data_math = PMD.iseng(data) ? transform_data_model(data) : data
        load_factor = generate_load_scenarios(data_math::Dict{String,<:Any}, N, ΔL)     # generate load scenarios
    return solve_robust_block_mld(data, load_factor, model_type, solver; kwargs...)
end
```